backend/src/main/java/de/neuefische/backend/appUser/AppUserRepository.java

```diff
@@ -7,4 +7,5 @@
 public interface AppUserRepository extends MongoRepository<AppUser, String> {
     Optional<AppUser> findByUsername(String username);
 
+    Optional<AppUser> findTopByOrderByAccountNumberDesc();
```

Comment:
Not a mistake, perhaps more a suggestion: find*Top*By... is not particularly easy to comprehend, so you might want to rename this to something self-explanatory?

Reply:
findTopByOrderByAccountNumberDesc is from libary. It will take  out the max value. In my case the max. accountNumber